<launch>
  <arg name="robot_namespace" default="robot"/>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find dataset_collector)/config/rec_model.yaml" command="load"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
	output="screen" args="joint_state_controller
					  joint1_position_controller"/>

</launch>